core/coins/format.go

```diff
@@ -10,53 +10,65 @@ import (
 	"cosmossdk.io/math"
 )
 
-// formatCoin formats a sdk.Coin into a value-rendered string, using the
-// given metadata about the denom. It returns the formatted coin string, the
-// display denom, and an optional error.
-func formatCoin(coin *basev1beta1.Coin, metadata *bankv1beta1.Metadata) (string, error) {
+// ConvertAmt converts given coin to `toDenom` base coin using metadata, and returns dec as string.
```

Comment:
@atheeshp why are there changes in this file? This touches format, so is not related to your PR, right?

Reply:
added this for some common code refactoring, reverted now.